remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingServer.java

```diff
@@ -128,6 +131,19 @@ public Thread newThread(Runnable r) {
                 }
             });
         }
+
```

Comment:
I am implementing a mechanism similar to [StartTLS](https://en.wikipedia.org/wiki/Opportunistic_TLS), this PR for now is in-progress.